client/components/flex-grid-column.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="flex-grid-column" :style="{ maxWidth: width }">
```

Comment:
Nit: Rename to 'flex-grid-item' to follow Base Web setup
https://baseweb.design/components/flex-grid/

Reply:
no problem